modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,6 +90,10 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Export SAML Metadata
+
+If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings to send to your identity provider. The identity provider can then upload these configuration settings to connect to your Anypoint Platform community.
```

Comment:
"Anypoint Platform community" seems kind of weird. I was emulating the SF language.

Reply:
Nope keeping it as *to connect to Anypoint Platform* is enough

Reply:
What about this:

"If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. Next, upload this file to your identity provider to configure Anypoint Platform."

Reply:
```suggestion
If your identity provider supports uploading service provider metadata, you can click the *Anypoint service provider metadata* link to download an XML file of your SAML configuration settings. You can then upload this file to your identity provider to configure Anypoint Platform.
```